### ISSUE TITLE
Plugins on Flatpak version

### ISSUE THREAD

**Issue by williamd1k0** [flathub/org.shadered.SHADERed#3]
Hi, I'm trying to run shadered with plugins, but I don't know how/where to place the plugin files.

I don't know if this is an issue with the flatpak version or the official one.

I tried to copy the files to the following dirs (without success):

- `$HOME/.var/app/org.shadered.SHADERed/`
- `$HOME/.var/app/org.shadered.SHADERed/data/`
- `$HOME/.local/share/shadered/`

**Comment by Calinou** [comment]
I don't think it works yet, but this *should* be fixed once the Flatpak is updated thanks to https://github.com/dfranx/SHADERed/pull/104.

**Comment by williamd1k0** [comment]
Okay, thanks!

**Comment by dfranx** [comment]
Apparently I forgot to make SHADERed to check for plugins in those directories. The fix for this will be included in v1.3.6

**Comment by williamd1k0** [comment]
So.. the latest flatpak version broke everything :P

![image](https://user-images.githubusercontent.com/10428035/87606552-a52ee580-c6d1-11ea-9034-e1a20c5872ef.png)

Also, trying to open/save anything does nothing now, the console output shows this error:
```
sh: xprop: command not found
```

**Comment by dfranx** [comment]
Ah, that's because it now looks for the file containing UI layout in a completely different location. You can reorder windows by yourself, it'll save the layout on exit + load it on the next startup.
Regarding this:
@Calinou Is there a way to copy a file to user's data directory during installation? Not really needed though, I could handle this issue programmatically.

As for the open/save bug, I'll have to look into that. All these bugfixes will be included in the next release (should be by the end of the next week).

**Comment by dfranx** [comment]
All three issues:
* messed up workspace
* not looking for plugins in correct directory
* file dialogs not working

should be fixed in the latest release 😀

**Comment by williamd1k0** [comment]
I reinstalled it and now got this error:
```
shadered: error while loading shared libraries: libSpvGenTwoLib.so: cannot open shared object file: No such file or directory
```
Version info:
```
flatpak list | rg -i shadered
SHADERed	org.shadered.SHADERed	1.3.6	stable	system
```
Am I doing something wrong?

**Comment by Calinou** [comment]
~~Maybe the CMake installation script isn't installing one of the required libraries?~~

**Edit:** Please continue the discussion about the missing library error here: https://github.com/flathub/org.shadered.SHADERed/issues/6

**Comment by frhun** [comment]
Just an idea i wanted to leave here: Maybe the most common extensions could be packaged as flatpak extensions. (see the kicad flatpak for an example of how that looks)
That way users can install the extensions they need right out of their software center.
This would potentially also have the useful side effect that these extensions would automatically support other hardware architectures.
